### PR TITLE
Add additional BelEID ATR-s

### PR DESCRIPTION
**Pull request by metsma**
Derive sign and auth algorithms from certificate

WE2-808

Signed-off-by: Raul Metsma <raul@metsma.ee>

### CODE REVIEW

File: src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp [hunk 1]
```diff
@@ -105,85 +103,62 @@ inline fs::path czechPkcs11ModulePath()
 #endif
 }
 
-const std::map<Pkcs11ElectronicIDType, Pkcs11ElectronicIDModule> SUPPORTED_PKCS11_MODULES {
+const std::map<ElectronicID::Type, Pkcs11ElectronicIDModule> SUPPORTED_PKCS11_MODULES {
     // EstEIDIDEMIAV1 configuration is here only for testing,
```
Comment (plaes) [inline src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp:107]: Comment is now out of sync as it still mentions `EstEIDIDEMIAV1`

Comment (metsma) [reply src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp]: Thank you

File: src/electronic-ids/ms-cryptoapi/MsCryptoApiElectronicID.cpp [hunk 2]
```diff
@@ -21,49 +21,60 @@
...
+byte_vector MsCryptoApiElectronicID::signWithAuthKey(const byte_vector& /* pin */,
+                                                     const byte_vector& hash) const
 {
-    const unsigned char* certPtr = cert.data();
-    auto x509 = SCOPE_GUARD(X509, d2i_X509(nullptr, &certPtr, long(cert.size())));
-    if (!x509) {
-        THROW(MsCryptoApiError, "Failed to create X509 object from certificate");
+    if (certType != CertificateType::AUTHENTICATION) {
+        THROW(WrongCertificateTypeError,
+              "This electronic ID does not support sigining with the authentication key. "
```
Comment (plaes) [inline src/electronic-ids/ms-cryptoapi/MsCryptoApiElectronicID.cpp:42]: Typo: `sigining`

Here and bunch of other locations, probably worth grepping for it in whole repo.

Comment (metsma) [reply src/electronic-ids/ms-cryptoapi/MsCryptoApiElectronicID.cpp]: Thanks

**Review by mrts — APPROVED** [review]

**Pull request merged**